From f6fdbbe989e54480882b9eeda452d168eea804a2 Mon Sep 17 00:00:00 2001
From: Daan De Meyer <daan.j.demeyer@gmail.com>
Date: Thu, 21 Dec 2023 19:46:54 +0100
Subject: [PATCH] Drop /dev test in test-mountpoint-util

Even /dev isn't always guaranteed to be a mount point, so let's drop
this part of the test.

(cherry picked from commit bacad14f94a4e98c3e81d821c56dbe7e2e4726ff)

Related: RHEL-30372
---
 src/test/test-mountpoint-util.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/src/test/test-mountpoint-util.c b/src/test/test-mountpoint-util.c
index 8555c5a7db..d8e56c0365 100644
--- a/src/test/test-mountpoint-util.c
+++ b/src/test/test-mountpoint-util.c
@@ -138,11 +138,6 @@ TEST(path_is_mount_point) {
         assert_se(path_is_mount_point("/proc/1/", NULL, AT_SYMLINK_FOLLOW) == 0);
         assert_se(path_is_mount_point("/proc/1/", NULL, 0) == 0);
 
-        assert_se(path_is_mount_point("/dev", NULL, AT_SYMLINK_FOLLOW) > 0);
-        assert_se(path_is_mount_point("/dev", NULL, 0) > 0);
-        assert_se(path_is_mount_point("/dev/", NULL, AT_SYMLINK_FOLLOW) > 0);
-        assert_se(path_is_mount_point("/dev/", NULL, 0) > 0);
-
         /* we'll create a hierarchy of different kinds of dir/file/link
          * layouts:
          *
